third_party/go9p/ufs_darwin.go

```diff
@@ -225,7 +225,7 @@ func (u *Ufs) Wstat(req *SrvReq) {
 			case true:
 				mt = st.ModTime()
 			default:
-				//at = time.Time(0)//atime(st.Sys().(*syscall.Stat_t))
+				// at = time.Time(0)//atime(st.Sys().(*syscall.Stat_t))
```

Comment:
code that is just commented out doesn't normally have spaces (unlike actual comments)